<launch>
  <node pkg="rl_navigation" type="controller.py" name="controller" output="screen"/>

  <node pkg="rl_navigation" type="ddpg_navi.py" name="ddpg_navi" output="screen">
    <param name="weight_path" value="trained_weight/ddpg_weights_actor.h5f"/>
    <param name="action_n" value="2"/>
  </node>


</launch>
